multiregion/scripts/run-consumer.sh

```diff
@@ -8,7 +8,7 @@ docker-compose exec broker-east-3 kafka-consumer-perf-test --topic multi-region-
     --broker-list broker-west-1:19091,broker-east-3:19093 \
     --timeout 20000
 
-echo -e "\n\n==> Consume from east: Multi-region Async Replication reading from Follower in east (topic: multi-region-async) \n"
+echo -e "\n\n==> Consume from east: Multi-region Async Replication reading from Observer in east (topic: multi-region-async) \n"
```

Comment:
@javabrett this change is good, but please make the corresponding change in the README as well